v2/internal/frontend/desktop/linux/window.go

```diff
@@ -48,7 +48,7 @@ static void SetMinMaxSize(GtkWindow* window, int min_width, int min_height, int
 GdkMonitor* getCurrentMonitor(GtkWindow *window) {
 	// Get the monitor that the window is currently on
 	GdkDisplay *display = gtk_widget_get_display(GTK_WIDGET(window));
-	GdkWindow *gdk_window = gtk_widget_get_window(GTK_WIDGET(window));
```

Comment:
This should be removed from this PR and put into a different PR and a different commit. In fact, this isn't even the change I meant to introduce (however, there is a bug in this code that I have a fix for).